packages/zowe-explorer/src/Profiles.ts

```diff
@@ -1313,6 +1303,23 @@ export class Profiles extends ProfilesCache {
         await vscode.window.showTextDocument(document);
     }
 
+    private async propmtToRefreshForProfiles(rootPath: string) {
```

Comment:
could the spelling be corrected here and the call to it `promptToRefreshForProfiles`

Reply:
Done.